pkg/dnfjson/dnfjson_test.go

```diff
@@ -887,3 +888,17 @@ echo '{"solver": "zypper"}'
 	assert.Equal(t, 0, len(res.Packages))
 	assert.Equal(t, 0, len(res.Repos))
 }
+
+func TestDepsolveResultWithModulesKey(t *testing.T) {
+	// quick test that verifies that `depsolveResult` understands JSON that contains
+	// a `modules` key
+	data := []byte(`{"modules": {}}`)
```

Comment:
I wonder how much value this test adds as it is.I personally would either omit it or expand it to cover some real world module example (in the sense of tests as documentation). Fine in a followup of course if we should get this in ASAP

Reply:
I'm going to prefer to do that as a followup. I mostly used this test case to have a very minor smoke test that the depsolver no longer breaks when a `modules` key is present.

I'll be updating to an idiom much like what you implemented in `osbuild` in https://github.com/osbuild/osbuild/pull/1993